src/vs/workbench/parts/files/electron-browser/fileActions.ts

```diff
@@ -725,6 +722,42 @@ class BaseDeleteFileAction extends BaseFileAction {
 			});
 		});
 	}
+
+	private getMoveToTrashMessage(distinctElements): string {
```

Comment:
@pradeepmurugesan the `distinctElements` is missing a type annotation: `distinctElements: ExplorerItem[]`

Reply:
fixed